tools/osv-linter/internal/checks/checks.go

```diff
@@ -40,6 +40,7 @@ type CheckDef struct {
 // Config defines the configuration for a check.
 type Config struct {
 	Verbose bool
+	Ecosystems []string
```

Comment:
can you add some comments to describe what these members mean exactly? It's not clear what specifying ecosystems in `Ecosystems` implies.

Reply:
Done in 6b9abed